NavigateU/NavigateU/HomeScreen/HomeDesignView.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  HomeDesignView.swift
+//  NavigateU
+//
+//  Created by Faki Doosuur Doris on 17.03.2024.
+//
+
+import UIKit
+
+class HomeDesignView: UIView {
+
+    lazy var appName: UILabel = UILabel()
+    lazy var slogan: UILabel = UILabel()
+    lazy var slogan2: UILabel = UILabel()
+    lazy var getStarted: UIButton = UIButton()
+    lazy var alreadyRegistered: UIButton = UIButton()
+    lazy var terms: UILabel = UILabel()
+    lazy var conditions: UILabel = UILabel()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setUpAppName()
+        setSlogan()
+        setSlogan2()
+        setUpGetstartedButton()
+        setUpAlreadyReg()
+        setUpTerms()
+        setUpConditions()
```

Comment:
lets make setupSubivews() function to avoid a lot of lines code in initialization